src/main/java/teammates/common/datatransfer/questions/FeedbackMsqQuestionDetails.java

```diff
@@ -46,8 +46,8 @@ public FeedbackMsqQuestionDetails() {
         this.msqChoices = new ArrayList<>();
         this.otherEnabled = false;
         this.generateOptionsFor = FeedbackParticipantType.NONE;
-        this.maxSelectableChoices = Integer.MIN_VALUE;
-        this.minSelectableChoices = Integer.MIN_VALUE;
+        this.maxSelectableChoices = 0;
+        this.minSelectableChoices = 0;
```

Comment:
@wkurniawan07 I have a doubt. 

I used 0 instead of Integer.MIN_VALUE to check if min/max choices are enabled. The reason is `Number.MIN_VALUE` used in the default structure also decimal numbers. So if the user does not select min/max checkbox, the backend will throw a `Server cannot process your request` error because of the decimal value.

I also considered using `Number.MIN_SAFE_INTEGER` however this calculates the min integer up to 53 bits, while 
` Integer.MIN_VALUE`  calculates the min integer up to 31 bits.

Reply:
`0` is a legitimate option value for min. selectable choices. Why not `-1`?

Reply:
I thought the minimum selectable choices must be greater than `0`.

```java
        if (isMinSelectableChoicesEnabled && minSelectableChoices < 1) {
            errors.add(Const.FeedbackQuestion.MSQ_ERROR_MIN_FOR_MIN_SELECTABLE_CHOICES);
        }
```

This is the existing backend validation for `FeedbackMsqQuestionDetails`

Reply:
@damithc has it always been the case that MSQ questions must have min. choice of 1?

This question:
```
Select programming languages that you know:
- Java
- C++
- ...
```
Looks like it should accept zero choice as an acceptable answer.

Reply:
```java
        if (isMinSelectableChoicesEnabled && minSelectableChoices < 1) {
            errors.add(Const.FeedbackQuestion.MSQ_ERROR_MIN_FOR_MIN_SELECTABLE_CHOICES);
        }
```
The min selectable choices can be 0 but if the checkbox is enabled, the min choices must be one as indicated by the validation

Reply:
The rationale is that specifying 0 as the minimum is the same as specifying there is no minimum number of options; in that case, the user can simply not enable the constraint at all.

Reply:
@RonakLakhotia that still does not change the fact that `0` is a value that can be legitimately set by user, not a value that should be specially treated by the system.

Reply:
Yes, so should we use`-1`? or some other integer like `-999`?

Reply:
I have suggested `-1` before.